install.sh

```diff
@@ -142,7 +142,7 @@ install_ubuntu() {
     if [[ "$container" != "docker" ]]; then
         apt install -y "linux-headers-$(uname -r)"
     fi
-    add-apt-repository -y ppa:wireguard/wireguard
```

Comment:
don't we need to add this repo anymore?

Reply:
Where from Wireguard packets comes from now?

Reply:
From test image (xenial ubuntu)

```console
# apt-cache policy wireguard
wireguard:
  Installed: 1.0.20200513-1~16.04.2
  Candidate: 1.0.20200513-1~16.04.2
  Version table:
 *** 1.0.20200513-1~16.04.2 500
        500 http://archive.ubuntu.com/ubuntu xenial-updates/universe amd64 Packages
        100 /var/lib/dpkg/status
# apt-cache policy wireguard-dkms
wireguard-dkms:
  Installed: 1.0.20200611-1ubuntu1~16.04.1
  Candidate: 1.0.20200611-1ubuntu1~16.04.1
  Version table:
 *** 1.0.20200611-1ubuntu1~16.04.1 500
        500 http://archive.ubuntu.com/ubuntu xenial-updates/universe amd64 Packages
        100 /var/lib/dpkg/status
# 

```

Reply:
Will this work for all Ubuntu versions, which we support: https://launchpad.net/~mysteriumnetwork/+archive/ubuntu/node-devnet ?

Reply:
Well we only test this script for:

- Xenial 16.04
- Bionic 18.04

The launchpad lists releases for 4 version:

- Xenial 16.04
- Bionic 18.04
- Eoan  19.10 (end of life 2020 Jul 17)
- Focal 20.04

![image](https://user-images.githubusercontent.com/20511486/95597294-1bd04a80-0a57-11eb-8cae-92f51d866304.png)

If I am reading this correctly we did not have a packaged released for Xenial since node version 0.31? So I question our support for it. Now `Eoan` was a transitional release (meaning no LTS).

The Bionic test pass without `add-apt-repository -y ppa:wireguard/wireguard`

So I might as well try adding Focal distro tests for good measure.

Reply:
I've added test for ubuntu 20.04